README.md

```diff
@@ -19,19 +19,19 @@ SSSD maintains two release streams - stable and LTM. Releases designated as
 LTM are long-term maintenance releases and will see bugfixes and security
 patches for a longer time than other releases.
 
-The list of all releases is maintained together with [SSSD documentation](https://sssd.github.io/users/releases.html).
+The list of all releases is maintained together with [SSSD documentation](https://sssd.io/users/releases.html).
 
 ## Building and installation from source
-Please see the [our developer documentation](https://sssd.github.io/developers/).
+Please see the [our developer documentation](https://sssd.io/developers/).
```

Comment:
404 again, this looks to be valid at the moment: https://sssd.io/docs/developers/